kolibri/plugins/learn/assets/src/vue/learn-page/index.vue

```diff
@@ -13,10 +13,10 @@
 
   <div class='button-wrapper'>
     <button @click='toggle()' v-if='expanded'>
-      <i class="material-icons">&#xE5CE;</i> Show Less
+      <i class="material-icons">&#xE5CE;</i> <span class="button-text">Show Less</span>
     </button>
     <button class='pure-button' @click='toggle()' v-else>
-      <i class="material-icons">&#xE5CF;</i> <span>Show More</span>
+      <i class="material-icons">&#xE5CF;</i> <span class="button-text">Show More </span>
```

Comment:
oh - I didn't see that in the global styles. Hmm I don't love the idea of putting that class there... I need to think about that a moment